internal/env.go

```diff
@@ -25,6 +27,7 @@ func BoolEnv(key string, def bool) bool {
 func IntEnv(key string, def int) int {
 	v, err := strconv.Atoi(os.Getenv(key))
 	if err != nil {
+		log.Warn("Non-integer value for env var %s, defaulting to %d. Parse failed with error: %v", key, def, err)
```

Comment:
I think this is a good change. Thanks for making it. Do you think we should do it for `BoolEnv` too?